var/spack/repos/builtin/packages/libressl/package.py

```diff
@@ -16,14 +16,16 @@ class Libressl(AutotoolsPackage):
 
     maintainers("eschnett")
 
+    version("3.7.2", sha256="b06aa538fefc9c6b33c4db4931a09a5f52d9d2357219afcbff7d93fe12ebf6f7")
+    version("3.6.3", sha256="87b1bbe36e9eec8d0ae5f04c83d36b2c5b0e581784c7eb0817025ed29eadea37")
     version("3.6.1", sha256="acfac61316e93b919c28d62d53037ca734de85c46b4d703f19fd8395cf006774")
 
     variant("shared", default=True, description="Build shared libraries")
     variant("static", default=False, description="Build static libraries")
 
     def configure_args(self):
         args = [
-            "--enable-shared" if "+shared" in spec else "--disable-shared",
-            "--enable-static" if "+static" in spec else "--disable-static",
+            "--enable-shared" if "+shared" in self.spec else "--disable-shared",
+            "--enable-static" if "+static" in self.spec else "--disable-static",
```

Comment:
If interested, it might clean up the package a bit to use `self.enable_or_disable()` similar to [in this package](https://github.com/spack/spack/blob/4fd7fa5fc1ddcb994348718e8e5cc3ba80ff8773/var/spack/repos/builtin/packages/dlb/package.py#L42C1-L42C1).